ejercicios.js

```diff
@@ -66,7 +66,7 @@ const capicuaNumber = (number) => {
 
 //Decimo-primer ejercicio: Programar una funcion que calcule el factorial de un numero n eje (5) = 120
 
-const factorial = (number) => {
```

Comment:
cuando solo pide un param, no es necesario usar parentesis

Reply:
Got it, cosas que uno olvida. Lo tendré mas pendiente.